src/atoms/Text/Text.stories.js

```diff
@@ -19,4 +19,22 @@ storiesOf('Text', module)
     <Text size={select('Size', sizeOptions, 'large')}>
       {text('Display Text', 'The quick brown fox jumps over the lazy dog')}
     </Text>
+  ))
+  .add('as span', () => (
+    <Text size={select('Size', sizeOptions, 'large')} as="span">
+      {text('Display Text', 'The quick brown fox jumps over the lazy dog')}
+    </Text>
+  ))
+  .add('show content in only 3 line and hide the rest ', () => (
+    <Text
+      size={select('Size', sizeOptions, 'large')}
+      truncate
+      maxLines={select('maxLines', 'number of lines', 3)}
+      _lineHeight="medium"
```

Comment:
Don't use _lineHeight. line height needs to be derived from size.

Reply:
done